tests/test_operation.py

```diff
@@ -1457,3 +1457,11 @@ def test_cry(self):
             ]
         )
         assert np.allclose(derivative, expected_derivative)
+
+        
+class TestValidationCheck:
+    """Tests for Validation Check for arguments passed to qml.operation.Observable"""
+
+    def test_validation():
+    with pytest.raises(Exception, match="Can only access when arguments are passed."):
+      qml.PauliZ()
```

Comment:
Hi @mayap24, it seems that something is going wrong with the indentation that causes tests to fail:
https://github.com/PennyLaneAI/pennylane/pull/1539/checks?check_run_id=3354356772#step:10:41

Hopefully, this should help. Also, adding a changelog entry would be required for the PR. See the [Submitting a Pull Request](https://pennylane.readthedocs.io/en/stable/development/guide/pullrequests.html) for more details. Let us know if the PR would be ready for review!
```suggestion
    def test_validation():
        with pytest.raises(Exception, match="Can only access when arguments are passed."):
            qml.PauliZ()
```